command_line.go

```diff
@@ -26,6 +26,7 @@ func ParseCommandLine(app *kingpin.Application) (*cmd.CommandLine, error) {
 	// 2. Parse flags
 	cmdLine := cmd.CommandLine{}
 	app.Flag("config", "path to rsc config file").Short('c').Default(path.Join(os.Getenv("HOME"), ".rsc")).StringVar(&cmdLine.ConfigPath)
+	app.Flag("retry", "number that rsc would retry on non-successful API response").Short('R').Default("0").IntVar(&cmdLine.Retry)
```

Comment:
Now that I am looking closer, can we change the short flag to something else? We already use a lower case 'r' for the refresh token and having this be an uppercase 'R' seems like it could cause confusion.  Maybe go with 't'?

Also, can you be more explicit in this as to what exactly is being retried.  So maybe something like "Number of retry attempts for non-successful API responses including authentication requests and response timeouts" (and anything else you can think of that is being retried, if any)